spec/pqueue_test.js

```diff
@@ -0,0 +1,94 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PriorityQueue from '../src/pqueue'
+
+chai.use(chaiChange)
+
+describe('PriorityQueue', () => {
+
+  it('exists', () => {
+    expect(PriorityQueue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
+    it('adds an element with priority (number) to the back of the queue.', () => {
+      const pQueue = new PriorityQueue()
+      expect(() => pQueue.enqueue('a', 5))
```

Comment:
These `expect`ations do not test the assertion made above (that `PriorityQueue` adds an element with a priority to the back of the queue).  You're testing that the size of the queue changes, but not the location that the element is inserted.